src/Compiler/Service/ServiceParsedInputOps.fsi

```diff
@@ -53,6 +62,9 @@ type public CompletionContext =
     /// or a single case union without a bar (type SomeUnion = Abc|)
     | TypeAbbreviationOrSingleCaseUnion
 
+    /// Completing a match expression
+    | Match of context: MatchContext
```

Comment:
Not right now. Adding support for `MatchLambda` is a good idea and should only involve an extra line of code.

What name would you then suggest instead of `Match` and `MatchContext`? Or is it the xml doc you're concerned about?

Reply:
`Match` is ok for now